### ISSUE TITLE
Feature Request support mermaid diagrams? 

### ISSUE THREAD

**Issue by ArEnSc** [HappenApps/Quiver#765]
https://knsv.github.io/mermaid/#mermaid

**Comment by cocoaaa** [comment]
+1:  This would be really helpful!

**Comment by egel** [comment]
Yes I agree, adding mermaid to Quiver would be an awesome feature especially I have lot of developer diagrams and this would be very helpful for me to be able to document all in here.

@ylian is there anything I could help you with to be able to push this feature further? 😃

**Comment by resting** [comment]
Started to draft diagrams with mermaid recently, have started to experiment using vscode as my main editor which has great support for most dev stuffs.

I do miss the block level editing though, and maybe a better search rather than vscode's text search.
